app.rb

```diff
@@ -132,7 +132,7 @@
 # This works for humans (i.e., browsers parse and follow the redirect) but,
 # because wget simply fetches the HTML document, this lets us continue to
 # spider the site to generate the contents of everypolitician/viewer-static.
-# See scripts/release.sh (update_viewer_static).
+# See scripts/release.sh (build_viewer_static and deploy_viewer_static).
```

Comment:
I think we only really need to be pointing at `build_viewer_static` here, as that's the part that's really relevant to this. This used to point at the whole method because that's all there was, but now that it's split, we can just point to the `build` part.